compiler/rustc_resolve/src/imports.rs

```diff
@@ -1371,9 +1369,9 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
             self.lint_buffer.buffer_lint_with_diagnostic(
                 UNUSED_IMPORTS,
                 id,
-                import.span,
+                import.use_span,
```

Comment:
`import.span` was the right span to use here.

Reply:
fixed.